getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,70 @@
+# Data Node Authentication
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes for the
+fragments. This means that the same role need to exist on the data
+nodes and authenticate to them.
+
+In a typical setup, you will deploy the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that we need to ensure that a user that is connected to the access
+node can access the same databases on the data nodes.
+
+## Setting up password authentication
+
+In order to set up password authentication for a user `new_user`, it
+is necessary to update the HBA (Host-Based Authentication) file,
+typically named `pg_hba.conf`, on the data nodes to enable password
+authentication for `new_user`. Details of the format of `pg_hba.conf`
+can be found in the [PostgreSQL manual, Chapter 20.1][postgresql-hba].
+
+In order to give `new_user` access to all databases on the data node,
+add the following line to the `pg_hba.conf` on the data node. This
+will use SCRAM SHA-256 password authentication. For other password
+authentication methods, see [PostgreSQL manual, Chapter
+20.5][auth-password].
+
+```
+host    all    new_user    .example.com    scram-sha-256
+```
+
+The assumption is that all the nodes are in the =*.example.com=
+domain.
+
+### Setting up passwords
+
+Passwords are not passed on the terminal but are added to a special
+password file. The default name of the password file is `passfile`
+located in the data directory for the PostgreSQL server, but it be set
+in the `postgresql.conf` file for the access node using the option
+`timescaledb.passfile`. You also want to set the name of the HBA file
+using the `hba_file` option:
+
+```
+timescaledb.passfile = pgpass.conf
+hba_file = pg_hba.conf
+```
+
+**Password file format.* The password file format is described in
+[Chapter 34.15 in the PostgreSQL manual][passfile-format] and consists
```

Comment:
Again, I don't think we should reference chapter numbers here. It is self-evident when clicking the link and chapter numbers might change.